spire-server.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: spire-server
+  version: 1.5.3
+  epoch: 0
+  description: The SPIFFE Runtime Environment (SPIRE) server
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+      - build-base
+      - git
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/spiffe/spire/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 08a8f97771afae82024b3b407ac4dafe284cd8658ac02855141607b6508b6da6
+  - runs: |
+      make bin/spire-agent
+      make bin/spire-server
+      install -Dm755 ./bin/spire-server "${{targets.destdir}}/usr/bin/spire-server"
+      install -Dm755 ./bin/spire-agent "${{targets.destdir}}/usr/bin/spire-agent"
+  - uses: strip
+subpackages:
+  - name: spire-agent
+    description: The SPIFFE Runtime Environment (SPIRE) agent
+    pipelines:
```

Comment:
this needs to be `pipeline:`

Reply:
Yep, it worked that way 🤦🏻‍♂️ thx so much. Can you please take a look at my other PR in melange?

https://github.com/chainguard-dev/melange/pull/238